.rosinstall

```diff
@@ -3,3 +3,12 @@
 # IF YOU WANT TO CHANGE THE ROS ENVIRONMENT VARIABLES
 # USE THE rosinstall TOOL INSTEAD.
 # IF YOU CHANGE IT, USE rosinstall FOR THE CHANGES TO TAKE EFFECT
+- setup-file: {local-name: /opt/ros/kinetic/setup.sh}
+- git: {local-name: /usr/share/ros/phidgets_api, uri: 'https://github.com/ros-drivers-gbp/phidgets_drivers-release.git',
+    version: release/kinetic/phidgets_api/0.7.5-0}
+- git: {local-name: /usr/share/ros/sicktoolbox, uri: 'https://github.com/ros-drivers/sicktoolbox.git'}
+- git: {local-name: /usr/share/ros/sicktoolbox_wrapper, uri: 'https://github.com/ros-drivers/sicktoolbox_wrapper.git'}
+- git: {local-name: /usr/share/ros/zed-ros-wrapper, uri: 'https://github.com/stereolabs/zed-ros-wrapper.git'}
+- git: {local-name: /usr/share/ros/ros_arduino_bridge, uri: 'https://github.com/hbrobotics/ros_arduino_bridge.git'}
+- git: {local-name: /usr/share/ros/razor_imu_9dof, uri: 'https://github.com/KristofRobot/razor_imu_9dof.git'}
+- git: {local-name: /usr/share/ros/nmea_navsat_driver, uri: 'https://github.com/ros-drivers/nmea_navsat_driver.git'}
```

Comment:
For several of these repositories, we have our own fork with a few changes. You should be able to the repo's we're using if you look in the `.gitmodules` file in the top level of the repo.

Reply:
I'll take a look at our own repo